requre/base_testclass.py

```diff
@@ -33,3 +33,19 @@ def setUp(self):
     def tearDown(self):
         PersistentObjectStorage().dump()
         super().tearDown()
+
+
+class RequreTestCaseOuterKeys(RequreTestCase):
+    """
+    This is base unittest TestCase what will help you to store requre storage files
+    It will create directory structre test_data/TEST_FILE_NAME in a directory where
+    test is stored. inside this subdir. there is created storage file via calling
+    self.id()
+    suffixed by yaml
+
+    Using Simplier Storage keys (avoid to store any module in CWD)
```

Comment:
these few lines focus on implementation details rather than saying 1) how to utilize this class 2) what are the benefits over the other one